tests/testAPI.sh

```diff
@@ -21,6 +21,29 @@ KRUIZE_URL="http://localhost"
 KRUIZE_PORT="31313"
 AUTH_TOKEN=""
 
+function usage() {
+	echo "Usage: $0 [-u kruize_url] [-t kruize_port] [-a auth_token]"
+	exit -1
+}
+
+# Iterate through the commandline options
```

Comment:
This entire block should be moved below all the existing functions, ie, just above the line https://github.com/kruize/kruize/blob/2a27733eed20987b0e9249fecbf95907d4615689/tests/testAPI.sh#L71

Reply:
Moved curl_cmd as well to avoid further issues. Thanks!!